packages/OpenSqueakMap.package/OSMChunkManager.class/instance/changed.st

```diff
@@ -0,0 +1,9 @@
+events
+changed
```

Comment:
Ich sehe gerade, dass sich das Entfernen des Viewports als Parameter durch viele Methoden hindurch zieht. 

Die Frage, ob die Multi-Instanz-Fähigkeit „ein ChunkManager – viele Viewports“ durch die Änderungen verloren geht, bezieht sich also auf alle in dem PR enthaltenen Änderungen.